rfc/002-log-odds-score-threshold.md

```diff
@@ -0,0 +1,115 @@
+
+#### Limitations
+* See above **Motivation** (e.g., **More Granular Similarity Metric**, **Low-*n* Person Clusters**, **User-Friendly**).
+
+### NBS Approach: Apply a User-Configurable Range the [Log Odds Score](https://cdc.sharepoint.com/:w:/r/teams/OPHDST-IRD-DIBBs/_layouts/15/Doc.aspx?sourcedoc=%7BCD3D39E1-FC95-4555-8841-15AE92ED317C%7D&file=DIBBs%20Record%20Linkage%20Algorithm%20and%20Terminology.docx&action=default&mobileredirect=true) Threshold (i.e. [`true_match_threshold`](https://github.com/CDCgov/RecordLinker/blob/c07aea2f15b6a86f54bcf48a37dcc5fac44d3ba4/src/recordlinker/assets/initial_algorithms.json#L102))
+*Patient-Patient Level*
+
+&nbsp;&nbsp;&nbsp;&nbsp;Given a Log Odds Score Threshold Range, [X, Y],
+
+&nbsp;&nbsp;&nbsp;&nbsp;When comparing incoming Patient record _i_ to existing Patient record _j_, if the sum of the log odds scores is...
+* < X, return **No Match**
+* \> Y, return **Match**
+* \>= X and <= Y, return **Possible Match (Manual Review)**
+
+#### Motivation
+* This approach would allow for more granular review of Possible Matches (upon match decision between a Patient record-Patient record pair, rather than upon Cluster Membership decision).
+* This is the approach [NBS](https://cdc.sharepoint.com/teams/CSELS-DHIS-ISB-NBS/_layouts/15/stream.aspx?id=%2Fteams%2FCSELS%2DDHIS%2DISB%2DNBS%2FShared%20Documents%2FNBS%20Modernization%20Project%2FDeDuplication%2FUX%2FPatient%20merge%20configuration%2Emov&referrer=StreamWebApp%2EWeb&referrerScenario=AddressBarCopied%2Eview%2Eb1b822b6%2Db97a%2D4550%2Db841%2Df79062d17baa) initially planned to take.
+
+#### Limitations
+* This approach implements Possible Match at a *pass level* (i.e., Manual Review of individual Patient record-Patient record match decisions), rather than a *Person level* (i.e., Manual Review of overall Person Cluster Membership decisions). As a result, there were limitations around how this approach would scale (e.g., Person Clusters with *n*>2 Patient records). For example, it is unclear how to determine whether an incoming Patient record matches a Person Cluster. Would this always be blocked by users completing Manual Review of all Possible Matches? Additionally, if a Patient record is pending Manual Review, will it be considered for comparison against future incoming Patient records in the queue? See [#18](https://github.com/CDCgov/RecordLinker/issues/18#top) for Patient record examples that demonstrate the limitations of this approach.
+* Implementing Possible Match (and thus Manual Review) at a *pass level* can create additional manual work. This manual work may also block the Record Linkage algorithm from determining a match decision, which can create bottlenecks for required Manual Reviews as the MPI grows. To arrive at a match decision, the user may need to reconcile many Possible Matches between different permutations of record pairs.
```

Comment:
I'm a bit unclear as to what the proposal is here for passes.  Are you proposing we only have 1 evaluation pass per algorithm going forward?